packages/lesswrong/components/users/ViewSubscriptionsPage.tsx

```diff
@@ -217,6 +218,14 @@ const ViewSubscriptionsPage = ({classes}: {
       noSubscriptionsMessage="You are not subscribed to any tags."
     />
     
+    {allowSubscribeToSequencePosts && <SubscriptionsList
+      title="Subscribed to Sequences"
+      collectionName="Sequences"
+      subscriptionType="newSequencePosts"
+      fragmentName="SequencesPageTitleFragment"
+      renderDocument={(sequence: SequencesPageTitleFragment) => <Link to={sequenceGetPageUrl(sequence)}>{sequence.title}</Link>}
+      noSubscriptionsMessage="You are not subscribed to any sequences."
+    />}
```

Comment:
I might clean up this page in a separate PR, to move away from using "subscription" since that can be confusing